internal/validations/observability/outputs/otlp.go

```diff
@@ -0,0 +1,17 @@
+package outputs
+
+import (
+	internalcontext "github.com/openshift/cluster-logging-operator/internal/api/context"
+	"github.com/openshift/cluster-logging-operator/internal/constants"
+	"strings"
+)
+
+func ValidateOtlpAnnotation(context internalcontext.ForwarderContext) (results []string) {
```

Comment:
my recollection is returning nothing means its valid, which means if found and if enabled... will fail

Reply:
It returns an empty array like the other one and appends it to the messages.      Validation only fails when len(messages) > 0, so this works afaik.

Reply:
Doesn't this fall through when the annotation is missing which means it will return:
```
return []string{"OTLP output annotation not found"}
```

Reply:
That's correct.   If the annotation is missing, we want validation to fail, so we add a message.     The condition on line28 below is set only when there are messages populated... 
`internalobs.NewConditionFromPrefix(obsv1.ConditionTypeValidOutputPrefix, out.Name, false, obsv1.ReasonValidationFailure, strings.Join(messages, ",")))`
If messages is empty, we set "output is valid".    You wrote the same thing for ValidateCloudWatchAuth().